client/src/views/Login.jsx

```diff
@@ -1,17 +1,40 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import { Container, Row, Col } from 'react-bootstrap';
 
 import LoginForm from '../components/LoginForm.jsx';
 
-export default function Login({ history }) {
-  return (
-    <Container className="pt-4">
-      <h1>Log In</h1>
-      <Row>
-        <Col className="col-4 pt-4">
-          <LoginForm onFinish={() => history.push('/profile-home')} />
-        </Col>
-      </Row>
-    </Container>
-  );
+class Login extends React.Component {
+  onLogin() {
```

Comment:
could you add a guard to verify that `this.props.profile` exists

Reply:
Done